### ISSUE TITLE
vscode launches a non-user instance of Chrome and can't connect to a local web service

### ISSUE THREAD

**Issue by u84six** [microsoft/vscode-chrome-debug#1005]
- VS Code Version:
1.44.2

Steps to reproduce:
I'm trying to debug an angular 8 app that should be able to access a dotnet core web api instance running locally on my machine (macOS). The error I see in the console of the angular app is **ERR_CERT_COMMON_NAME_INVALID** after trying to make the first call to the local web api service. I have a dev cert setup and works when not debugging.

1. Start dotnet core web api service: dotnet run
2. Start angular 8 app web server: ng serve --configuration=local --disable-host-check
3. Run vscode debugger with launch config:
`{
    "type": "chrome",
    "request": "launch",
    "name": "Launch Chrome localhost,
    "url": "http://localhost:4200",
     "webRoot": "${workspaceFolder}/frontend" //this is the path to the app root (angular)
 }`
4. Chrome opens and fails to access local web api service with the above error

Here's the interesting thing. If I open a new tab in my current instance of Chrome (i.e. the instance I'm logged into that has all my extensions etc), I'm able to access the local web api service using the same exact url http://localhost:4200. The problem happens only with the Chrome instance that the Chrome debugger extension launches, which for some reason, is a stripped-down bare instance of Chrome (i.e. has none of my extensions loaded, etc). I think this might be the key to the reason that it's not working for me. How is the debugger launching Chrome? Why doesn't it have my extensions, etc? Why does the instance that the debugger launches not connect to the local web service, yet the instance of Chrome that I already have running can?

**Comment by u84six** [comment]
I figured out the issue. The instance of Chrome that the extension launches required explicit confirmation to access the non-secure website (in this case, my local web api running on kestrel ssl). After forcing the browser to "access anyway", I was able to connect to the web service. The reason this threw me off was because of the 2 different instances of Chrome. The instance I use to browse already has access to the local web api service, while the instance that the extension launches did not. I'm curious why the extension doesn't just launch Chrome the same way it's opened by the user. This would be helpful because then I'd be able to use dev extension I already have installed. Anyhow, problem resolved.
